packages/maskbook/src/plugins/BarnBridge/Import/BarnBridgeFrontEnd/PortfolioBalance.tsx

```diff
@@ -0,0 +1,97 @@
+import { makeStyles } from '@masknet/theme'
+
+import { Typography, Divider, LinearProgress } from '@material-ui/core'
+import { formatUSDValue } from './web3/utils'
+import { useI18N } from '../../../../utils/i18n-next-ui'
+import { COLOR_SY_SENIOR_TEXT, COLOR_SY_JUNIOR_TEXT } from '../../constants'
+
+const s = makeStyles()((theme) => ({
+    progress: {
+        display: 'flex',
+        marginTop: 24,
+    },
+    dataColumn: {
+        paddingLeft: 16,
+        position: 'relative',
+        backgroundColor: 'var(--color)',
+        borderRadius: '50%',
+        height: 8,
+        left: 0,
+        top: 4,
+        width: 8,
+    },
+    portfolioAmountContainer: {
+        display: 'flex',
+        padding: 10,
+        paddingLeft: 30,
+        paddingRight: 30,
+        justifyContent: 'space-between',
+        alignSelf: 'space-between',
+        alignContent: 'space-between',
+        alignItems: 'space-between',
+        flex: 1,
+        flexDirection: 'row',
+    },
+}))
+
+type PortfolioPropsLabel = [label: string, value: number]
```

Comment:
You can just define it as [Tuple](https://www.typescriptlang.org/docs/handbook/2/objects.html#tuple-types).